lnd.go

```diff
@@ -227,11 +227,10 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 	// Enable http profiling server if requested.
 	if cfg.Profile != "" {
 		go func() {
-			listenAddr := net.JoinHostPort("", cfg.Profile)
 			profileRedirect := http.RedirectHandler("/debug/pprof",
 				http.StatusSeeOther)
 			http.Handle("/", profileRedirect)
-			fmt.Println(http.ListenAndServe(listenAddr, nil))
+			fmt.Println(http.ListenAndServe(cfg.Profile, nil))
```

Comment:
We should add a log here to show the final interface we end up listening on. May be useful in debugging why things aren't showing up.

Reply:
Added log